src/HW2.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Scanner;
+
+public class HW2 {
+
+    public static void homeWorkTask(int a) {
+        a = 555;
+        int sumOfThree = 5 + 5 + 5;
+    }
+
+    public static void homeWorkTask(String name, String lastname) {
+        name = "John";
```

Comment:
The same - every method in Java should be general and universal, your method works with "John" and "Smith".

    static void ourMethod(String word1, String word2) {
        String concatWords = word1 + word2;
        int lengthOfContactWords = concatWords.length();
        System.out.println(lengthOfContactWords);
    }

We use ourMethod in the main method:
ourMethod("John", "Smith");